From 70a12864178396592d26883bc404f15884fb44a0 Mon Sep 17 00:00:00 2001
From: taaem <taaem@mailbox.org>
Date: Sat, 10 Dec 2016 13:26:02 +0100
Subject: [PATCH] Disable building of AppImages and debs By default the
 packages.json in the source specifies AppImage and deb as target but since we
 don't need them disable them and only build into the out dir.

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 402a477..bffc9e0 100644
--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
       "afterRemove" : "bin/deb/after-remove.tpl",
       "category": "Network",
       "depends": ["libappindicator1", "libasound2", "libgconf-2-4", "libnotify-bin", "libnss3", "libxss1"],
-      "target": ["AppImage", "deb"]
+      "target": ["dir"]
     }
   },
   "devDependencies": {
-- 
2.11.0

